static/app/views/issueDetails/traceTimeline/useTraceTimelineEvents.tsx

```diff
@@ -17,8 +17,8 @@ interface BaseEvent {
   transaction: string;
 }
 
-interface TimelineDiscoverEvent extends BaseEvent {}
-interface TimelineIssuePlatformEvent extends BaseEvent {
+interface TimelineIssuePlatformEvent extends BaseEvent {}
+interface TimelineDiscoverEvent extends BaseEvent {
```

Comment:
This interface should have had `event.type` and `stack.function` from the get-go.